server_installation/topics/clustering/troubleshooting.adoc

```diff
@@ -1,19 +1,22 @@
 
 === Troubleshooting
 
-Note that when you run cluster, you should see message similar to this in the log of both cluster nodes:
-
+* Note that when you run cluster, you should see message similar to this in the log of both cluster nodes:
++
 [source]
 ----
 INFO  [org.infinispan.remoting.transport.jgroups.JGroupsTransport] (Incoming-10,shared=udp)
 ISPN000094: Received new cluster view: [node1/keycloak|1] (2) [node1/keycloak, node2/keycloak]
 ----
 If you see just one node mentioned, it's possible that your cluster hosts are not joined together.
-
++
 Usually it's best practice to have your cluster nodes on private network without firewall for communication among them.
 Firewall could be enabled just on public access point to your network instead.
 If for some reason you still need to have firewall enabled on cluster nodes, you will need to open some ports.
 Default values are UDP port 55200 and multicast port 45688 with multicast address 230.0.0.4.
 Note that you may need more ports opened if you want to enable additional features like diagnostics for your JGroups stack.
 {project_name} delegates most of the clustering work to Infinispan/JGroups.
 For more information, see link:{appserver_jgroups_link}[JGroups] in the _{appserver_jgroups_name}_.
+
+* If you are interested about failover support (hight availability), evictions, expiration and cache tunings, you can go to the
+<<cache-configuration>>.
```

Comment:
* If you are interested in failover support (high availability), evictions, expiration and cache tuning, see ...